CLAUDE.md

```diff
@@ -0,0 +1,85 @@
+# CLAUDE.md
```

Comment:
thank you for this, the task to add this is on me but got wrapped up in other s*** 🍻